extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -106,7 +106,9 @@ class AmpStickyAd extends AMP.BaseElement {
     if (scrollTop > viewportHeight) {
       this.removeOnScrollListener_();
       this.deferMutate(() => {
-        toggle(this.element, true);
+        setStyles(this.element, {
+          'visibility': 'visible',
+        });
         this.viewport_.addToFixedLayer(this.element);
         this.scheduleLayout(this.ad_);
```

Comment:
You'll have to claim ownership of the `<amp-ad>`, so that `Resources` no longer schedules its lifecycle leaving it to the `<amp-sticky-ad>`.